app/api/openai/processTranscript.ts

```diff
@@ -35,9 +35,9 @@ export async function processTranscript(): Promise<FormattedResponse> {
 """
 {
 "title": "curriculum meeting",
-"attendees": "John, Paula, Mike",
-"key discussion points" :  "Reviewed student progress and performance in the current cohort", "Discussed upcoming curriculum changes and improvements", "Addressed the need for additional support for struggling students",
-"actions":  "John to conduct weekly check-ins with struggling students", "Mike to email all students with schedule changes", "Paula to create a new curriculum proposal"
+"attendees": ["John", "Paula", "Mike"],
```

Comment:
I think this is a good improvement and will make handling these strings more dynamic in the future.